Intermediate/CreateExplicitMeasures.csx

```diff
@@ -0,0 +1,81 @@
+// Title: Auto-create explicit measures from all columns in all tables that have qualifying aggregation functions assigned 
+//  
+// Author: Tom Martens, twitter.com/tommartens68
+// 
+// This script, when executed, will loop through all the tables and creates explicit measure for all the columns with qualifying
+// aggregation functions.
+// The qualifiyng aggregation functions are SUM, COUNT, MIN, MAX, AVERAGE.
+// This script can create a lot of measures, as by default the aggregation function for columns with a numeric data type is SUM.
+// So, it is a good idea to check all columns for the proper aggregation type, e.g. the aggregation type of id columns 
+// should be set to None, as it does not make any sense to aggregate id columns.
+// An annotation:CreatedThrough is created with a value:CreateExplicitMeasures this will help to identify the measures createed
+// using this script.
+// What is missing, the list below shows what might be coming in subsequent iterations of the script:
+// - the base column property hidden is not set to true
+// - no black list is used to prevent the creation of unwanted measures
+
+// ***************************************************************************************************************
+//the following variables are allowing to control the script
+var overwriteExistingMeasures = 0; // 1 overwrites existing measures, 0 preserves existing measures
+
+var MeasureNameTemplate = "{0} ({1})"; // String.Format is used to create the measure name. 
+//{0} will be replaced with the columnname (c.Name), {1} will be replaced with the aggregation function.
+//Assuming the column name is "Sales revenue" and the aggregation function is SUM 
+//the new measure name will be: "Sales revenue (Sum)"
+
+//store aggregation function that qualify for measure creation to the array ArrayAggFunctions
+string[] ArrayAggFunctions = new string[5];
+ArrayAggFunctions[0] = "Sum";
+ArrayAggFunctions[1] = "Count";
+ArrayAggFunctions[2] = "Min";
+ArrayAggFunctions[3] = "Max";
+ArrayAggFunctions[4] = "Average";

```

Comment:
Since this is only used for checking, use a HashSet instead of an array. Also, no reason to convert everything to strings - let's just use the [AggregateFunction](https://docs.microsoft.com/en-us/dotnet/api/microsoft.analysisservices.tabular.aggregatefunction?view=analysisservices-dotnet) enum type directly. So line 27-32 should read:

```csharp
var aggFunctions = new HashSet<AggregateFunction>{
    AggregateFunction.Sum,
    AggregateFunction.Count,
    AggregateFunction.Min,
    AggregateFunction.Max,
    AggregateFunction.Average
};
```